client/src/components/CreatePost.test.tsx

```diff
@@ -3,7 +3,6 @@ import CreatePost from "./CreatePost";
 import { unmountComponentAtNode } from "react-dom";
 import { act } from "react-dom/test-utils";
 import { render, screen, fireEvent, cleanup } from "@testing-library/react";
-import newImage from "../assets/background.jpg";
 
 let container: HTMLElement | null = null;
```

Comment:
I don't believe this as well as the afterEach are necessary for the tests to work, you can remove them.